lib/conjur/command/hosts.rb

```diff
@@ -153,7 +159,7 @@ def self.host_layer_roles host
       end
     end
 
-    hosts.desc "[Deprecated] Enroll a new host into conjur"
+    hosts.desc "Enroll a new host into conjur [DEPRECATED]"
```

Comment:
Seems like the `action` block for this command should either call `notify_deprecated`, or have a comment explaining why it doesn't need to. Right now it looks like a mistake.